notebooks/COS/DataDl/DataDl.ipynb

```diff
@@ -160,7 +160,7 @@
     "\n",
```

Comment:
<p>It may be more effective to move the comments to a markdown cell above the code and provide explanation, followed by the code example. This would allow for a cleaner and more organized presentation of the code, while providing valuable insights for readers.</p><p> </p><p>Also, it may not be necessary to include an enumaration for the loop in code, as we do not appear to be utilizing the index other than the loop itself. Instead, we could achieve the same result with a simple for loop.</p><p> </p><pre class="ql-syntax" spellcheck="false">obsIdList = ['LDYR52010', 'LBNM01040', 'LBBD04040']
with open('./obsId_list.txt', 'w') as f:
    f.write("Dataset_id\n")
&nbsp;&nbsp;  for obsId in obsIdList[:-1]:
&nbsp;&nbsp;      f.write(obsId + "," + "\n")
&nbsp;  &nbsp;f.write(obsIdList[-1])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Thanks for the update. No need for the following line of code:</p><pre class="ql-syntax" spellcheck="false">obsIdList_length = len(obsIdList)
</pre>